src/commands/new.js

```diff
@@ -27,6 +27,12 @@ async function command (context) {
   // camelCase the project name for user example
   const projectNameCamel = upperFirst(camelCase(projectName))
 
+  // Guard against `ignite new ignite`
+  if (projectNameCamel === 'ignite') {
```

Comment:
I'm pretty sure this won't work (it'll be `Ignite` not `ignite`) ... can you test @fvonhoven ?